res/database/Default/database_scripts.xml

```diff
@@ -105,10 +108,10 @@
 					<en>Hilarious|Lovely|Turbulent|Romantic|Melancholic|Surprising</en>
 				</funny>
 				<namewoman>
-					<de>%PERSONGENERATOR_NAME(unk,2)%</de>
+					<de>${.persongenerator:"name":"":"female"}</de>
```

Comment:
In anderen Fällen wurde NAME hier durch firstname ersetzt (auch 3 Zeilen weiter). In der Expressionauswertung wird das auch so interpretiert, aber ich fände es für die Eindeutigkeit besser, auch in der Datenbank explizit zu sein.